### ISSUE TITLE
Support for skin temperature with Scubapro G2

### ISSUE THREAD

**Issue by p-neves** [subsurface/subsurface#1186]
### Describe the issue:

- [ ] Bug
- [ ] Change request
- [x] New feature request
- [ ] Discussion request

### Issue long description:
Is it possible to add support for the skin temperature info recorded by the Scubapro G2 computer?
This info is recorded during the dive via the heartband. It will add (yet) another button on the toggle bar, but...

### Subsurface version:

4.7.7-167

**Comment by dirkhh** [comment]
I do worry about diminishing returns vs. overload of the UI / profile.
Not saying "don't do this", just raising a concern...

**Comment by p-neves** [comment]
Yes. I know what you mean. The thing is that we can choose what to display by toggling the buttons on the bar... Although it will add another button on the bar...

**Comment by dje29** [comment]
+1

I could provide help with implementation and/or test/test data (I'm using G2 with Heart Rate Belt)

**Comment by satoschu** [comment]
Hi, the Integration of this would be great! I can also help to test it.
Best regards
